.travis.yml

```diff
@@ -42,6 +42,6 @@ deploy:
 notifications:
   email: false
 
-cache:
-  directories:
-  - $HOME/.m2
+#cache:
+#  directories:
+#  - $HOME/.m2
```

Comment:
Do we need to disable the cache for this?

Reply:
My theory is that the cache is what's causing the problem and this is the only way i can test it.

Reply:
ok, go for it in that case. Only issue, I see you disabled the build but also bumped the version. Should you do only one of the 2 things?